### PR TITLE
Port to store enhancer

### PR DESCRIPTION
**Pull request by samiskin**
As per @gaearon's suggestion, this PR turns this library into a store enhancer.

### TODOs

 - [x] Add comments
 - [ ] Update README
 - [x] Code review

### CODE REVIEW

**Comment by CharlieHess** [comment]
:gem:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-electron-store",
-  "version": "0.2.5",
+  "version": "0.3.0",
```
Comment (paulcbetts) [inline package.json]: It's really compatible with 0.2.x?

Comment (samiskin) [inline package.json]: It isn't but Charlie said it'd be better as 0.3.0 when its only 2 days old.

Comment (samiskin) [inline package.json]: Think it's good for 1.0.0 anyway?

**Pull request merged**